F/FMUComplianceChecker/build_tarballs.jl

```diff
@@ -0,0 +1,48 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "FMUComplianceChecker"
+version = v"2.0.4"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/modelica-tools/FMUComplianceChecker/archive/refs/heads/master.zip", "2f8f0754164c7c13f8de76f8ec3271899d4650a3ffe895c2e813cd0b46ae7cb7")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+apk add subversion
+mkdir ${bindir}/
+cd FMUComplianceChecker-*/   
+mkdir build; cd build
+cmake ..
```

Comment:
Please read documentation: https://docs.binarybuilder.org/stable/build_tips/#CMake-builds
```suggestion
cmake -DCMAKE_INSTALL_PREFIX=${prefix} -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release ..
```